wiki/History_of_osu!/2009/en.md

```diff
@@ -35,3 +35,128 @@ osu! makes its way to the iPhone from a port to the iPhone by nuudles. The proje
+
+## June
+
+Due to a change to the [Beatmap submission system](/wiki/Beatmapping/Beatmap_submission) users were no longer able to upload any modifications to a beatmap after it was modded. There were cases of beatmaps becoming ranked without the BAT member who ranked the beatmap knowing that it contained [rule-breaking changes](/wiki/Ranking_criteria).[^bss-update]
+
+Before, osu! was only available in English. With the new localisation system coming into place, users became able to translate the game client. They could download a file containing the game's text lines, modify it to their language and ask the file to be integrated into the game client.[^client-localisation][^stable-b695]
+
+## July
+
+On 14 July, osu! news became available as a [RSS feed](https://en.wikipedia.org/wiki/RSS).[^rss-news]
+
+Two updates this month brought a new mouse precision toggle option for low sensitivities as well as new command parameters and shortcuts to simplify [storyboard scripting](/wiki/Storyboard/Scripting) in osu!.[^stable-b753c][^stable-b709]
+
+The database was upgraded with an additional 32 GB SSD to support more storage going forward.[^server-upgrade]
+
+After a first initial test for Taiko online rankings was done in March, ranking statistics became viewable per mode on the osu! website.[^per-mode-ranking][^stable-b622] Prior to that, only the [osu! game mode](/wiki/Game_mode/osu!) had online rankings.
+
+## August
+
+The Twitter account [@osustatus](https://twitter.com/osustatus) was created, intended as a status page to inform about issues with [Bancho](/wiki/Bancho_(server)) and the website.[^osustatus]
+
+osu! received a new `LoopType` parameter for looping storyboard objects and a [Relax](/wiki/Gameplay/Game_modifier/Relax) mod for the Taiko game mode in an update on August 12. Additionally, [AiMod](/wiki/Client/Beatmap_editor/AiMod) was published in a pre-release version, a tool for detecting issues in beatmaps.[^stable-b825]
+
+## September
+
+Updates this month integrated new features, where among the changes were widescreen support for song select,[^stable-b904a] friend support on the website,[^custom-changelog] and the Taiko mascot [pippidon](/wiki/Mascots#pippi) would now be displayed in the top-left corner while playing.[^stable-b898]
+
+As of 6 September 2009, all [Taiko scores were reset](https://osu.ppy.sh/community/forums/topics/17118?n=1) to allow for an improved scoring algorithm to be used.
+
+## October
+
+A new ranking info screen was added which layed the groundwork for the later implemented ranking chart system.[^stable-b1077][^custom-changelog]
+
+[Larto](https://osu.ppy.sh/users/12328) published the beatmap [Imogen Heap - Sweet Religion](https://osu.ppy.sh/beatmapsets/10000#osu/97517), meaning that the total count of submitted beatmaps reached 10,000.[^beatmap-10k]
+
+In addition to osu! news in July 2009, the [changelog](/wiki/Changelog) became available as a RSS feed on 25 October too.[^rss-changelog]
```

Comment:
I'm not entirely sure how to handle RSS stuff, since the [Changelog](https://osu.ppy.sh/wiki/en/Changelog#history) article already has a history entry.